tools/i18n/create-legacy-stream-translations

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import json
+import os
+import re
+from subprocess import check_output
+from typing import Dict, List
+
+
+def get_json_filename(locale: str) -> str:
+    return f"locale/{locale}/translations.json"
+
+
+def get_locales() -> List[str]:
+    output = check_output(["git", "ls-files", "locale"], text=True)
+    tracked_files = output.split()
+    regex = re.compile(r"locale/(\w+)/LC_MESSAGES/django.po")
+    locales = []
+    for tracked_file in tracked_files:
+        matched = regex.search(tracked_file)
+        if matched and matched.group(1) != "en_GB":
+            locales.append(matched.group(1))
+
+    return locales
+
+
+def create_legacy_stream_translations(resource: str) -> None:
```

Comment:
This would only need to be done once, so we probably don't need to save this tool/script to the repository. But rather the person who creates `legacy_stream_translations.json` files with a version of this script would commit those files to the repository.